packages/patternfly-4/react-inline-edit-extension/src/components/InlineEdit/index.js

```diff
@@ -0,0 +1,4 @@
+export { TableEditConfirmation } from './constants';
+export { default as editableTableBody } from './BodyHOC';
+export { default as editableRowWrapper } from './RowWrapperHOC';
```

Comment:
I added HOC suffix to the Body and the RowWrapper. 

Errors were thrown by gatsby ( gatsby-transformer-react-docgen -> react-dcogen) when leaving just the RowWrapper. It may be a collision with the other RowWrapper or just that react-docgen doesn't really like HOC. I tried many things (e.g. custom resolver) and nothing worked. 

Just renaming is a simple solution.

other options could be
- rowWrapperHOC
- editableRowWrapper
- EditableRowWrapper

not sure which one is the best

Reply:
totally fine w/ "editableRowWrapper" if others are...

Reply:
ok, renamed to "editableRowWrapper"